### ISSUE TITLE
Animes are not playing again 

### ISSUE THREAD

**Issue by Manniga** [mukul500/AnimeXStream#463]
It worked for a few days then stopped working like before.

**Comment by The-BOT-says** [comment]
Yes it's stopped playing new episodes, but we can play previous episodes without any kind of lag.
Fix this issue

**Comment by wolbag** [comment]
The Animes Won't Play, I'm still curious if it is just Maintenance or just Technical difficulty

**Comment by wolbag** [comment]
> Use saikou until it's fixed if u want

What is Saikou? Where can i Download it?

**Comment by ITADORI-15** [comment]
@wolbag https://github.com/saikou-app/saikou

**Comment by wolbag** [comment]
> @wolbag https://github.com/saikou-app/saikou

Omg. I don't know you but you helped me alot, Thank you ITADORI san(. ❛ ᴗ ❛.)

**Comment by ITADORI-15** [comment]
Np bro

**Comment by GABIlopez** [comment]
Anime

**Comment by GABIlopez** [comment]
Anime  apk  android
